pkg/crypto/crypto_test.go

```diff
@@ -6,14 +6,12 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
-
-	"github.com/filecoin-project/go-shared-types/internal/crypto"
```

Comment:
I believe you still need to import crypto since this is a separate test package.

Reply:
Yep, you're right. I just flipped the import order to conform to our conventions.